sdk/tables/data-tables/swagger/README.md

```diff
@@ -35,3 +35,11 @@ directive:
     transform: >
       $["description"] = "A summary of request statistics grouped by API";
 ```
+
+```yaml
```

Comment:
Do we need to re-run the code generator after this change?  I would have expected some changes to the generated code due to this new directive.

Reply:
Yup good catch!